engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/history/batch/removaltime/SetRemovalTimeToHistoricBatchesDto.ftl

```diff
@@ -0,0 +1,22 @@
+<#-- Generated From File: camunda-docs-manual/public/reference/rest/history/batch/post-batch-set-removal-time/index.html -->
+<#macro dto_macro docsUrl="">
+<@lib.dto extends="AbstractSetRemovalTimeDto">
+
+    <@lib.property
+        name = "historicBatchQuery"
+        type = "object"
+        dto = "HistoricBatchQueryDto"
```

Comment:
This DTO is not yet defined in OpenAPI, as far as I can see, right? Do we need to define it?

Reply:
Wow, I was probably a bit too preoccupied with the summit. Indeed, I forgot to add that DTO. Good catch. Interesting that it compiles without it.